lib/src/base-styles/_nx-btn.scss

```diff
@@ -79,11 +79,11 @@
   }
 }
 
-.nx-btn--primary {
+.nx-btn--primary, .nx-btn--primary[href] {
```

Comment:
If these blocks were included in the above block using `&--primary` etc, we wouldn't need to repeat the new selector

Reply:
> If these blocks were included in the above block using `&--primary` etc, we wouldn't need to repeat the new selector

Great idea, much cleaner now: [eb5d38c](https://github.com/sonatype/sonatype-react-shared-components/pull/1137/commits/eb5d38c1ad7af033d1a7a25f66d6d734df3212ee)